README.md

```diff
@@ -262,6 +262,10 @@ specified first. Also, `--check` ignores the value of the `controller` and assum
 will be present during a real run. That means, in presence of an invalid `controller`,
 `--check` may signal success but the actual play run fails.
 
+When destroying and removing the bonding profiles, if only bringing down the
```

Comment:
Having both a "when..." and a "if ..." clause seems to be complicated and hard to understand. I don't understand this part, since it first mentions destroying and removing, then "bringing down". It would be best to simplify this as simple if, then sentences.